page.gohtml

```diff
@@ -11,6 +11,8 @@
 
     <title>{{.Title}}</title>
 
+    <link rel="icon" type="image/x-icon" href={{.Logo}}>
```

Comment:
Didn't notice this earlier, but looks like the quotes on the attribute value are missing after this generates?

Reply:
Fixed - good catch!